### ISSUE TITLE
help

### ISSUE THREAD

**Issue by yazdandeli** [EXALAB/AnLinux-App#216]
help me for install on galaxy a10 android pie

**Comment by exaksana** [comment]
## Not a Bug — Installation Help Request

Android Pie (9.0) is fully supported. Galaxy A10 uses an ARM chipset (Exynos 7884), compatible.

**Quick setup:**
1. Install **Termux** from F-Droid (NOT Play Store — it's deprecated):
   https://f-droid.org/packages/com.termux/
2. Install **AnLinux** from Play Store or GitHub Releases.
3. Open AnLinux → choose your distro → tap "Copy" on the commands → paste into Termux.
4. Wait for the rootfs download and extraction to complete.
5. Run the generated start script: `./start-ubuntu.sh` (or whichever distro).

**Minimum Termux setup before starting:**
```sh
pkg update && pkg upgrade -y
termux-setup-storage
```

If you run into specific errors, open a new issue with the exact error output. This issue is too vague to diagnose. Closing.

**Comment by exaksana** [comment]
@supersonictw plz close it
